wire-ios/Wire-iOS UnitTests/UserInterface/Conversation/Content/Cells/MLSMigrationCellDescriptionTests.swift

```diff
@@ -47,4 +47,94 @@ final class MLSMigrationCellDescriptionTests: XCTestCase {
         XCTAssertNil(cellDescription.accessibilityIdentifier)
         XCTAssertEqual(cellDescription.accessibilityLabel?.isEmpty, false)
     }
+
+    // MARK: - Attributed Strings
+
+    func test_mlsMigrationStarted_doesContainLinkInAttributedString() throws {
+        // given
+        let cellDescription = MLSMigrationCellDescription(messageType: .mlsMigrationStarted)
+        let expectation = self.expectation(description: "")
```

Comment:
Is this expectation really needed?
I assume the block submitted to `enumerateAttribute` is not dispatched but called synchronously.

Reply:
OMG you are right, thanks for the hint! 11dbc15f13c85251fd5ee15ef2a14e38279bfb56